README.md

```diff
@@ -83,6 +83,17 @@ Run the following commands to generate the libraries needed by Hermes:
 
 `make`
 
+4) Including the generated SimpleAmqpClient library
```

Comment:
I know you are following the what was there before, but this `4)` doesn't take advantage of the asciidoc syntax. Can make an issue and fix it in a separate PR?

It should be

```
. Asciidoc lists
. should use bullets
. like this
```